Code/Editor/MainWindow.cpp

```diff
@@ -2137,6 +2160,291 @@ bool MainWindow::focusNextPrevChild(bool next)
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.resources.forums", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnDocumentationForums();
+            }
+        );
+    }
+
+    // AWS Support
+    {
+        ActionProperties actionProperties;
+        actionProperties.m_name = "AWS Support";
+        actionProperties.m_category = "Help";
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.resources.awssupport", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnDocumentationAWSSupport();
+            }
+        );
+    }
+
+    // About O3DE
+    {
+        ActionProperties actionProperties;
+        actionProperties.m_name = "&About O3DE";
+        actionProperties.m_category = "Help";
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.abouto3de", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnAppAbout();
+            }
+        );
+    }
+
+    // Welcome
+    {
+        ActionProperties actionProperties;
+        actionProperties.m_name = "&Welcome";
+        actionProperties.m_category = "Help";
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.welcome", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnAppShowWelcomeScreen();
+            }
+        );
+    }
+
+}
+
+void MainWindow::InitializeMenus()
+{
+    // Initialize Menus
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&File";
+        m_menuManagerInterface->RegisterMenu(FileMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Edit";
+        m_menuManagerInterface->RegisterMenu(EditMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Game";
+        m_menuManagerInterface->RegisterMenu(GameMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Tools";
+        m_menuManagerInterface->RegisterMenu(ToolsMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&View";
+        m_menuManagerInterface->RegisterMenu(ViewMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Help";
+        m_menuManagerInterface->RegisterMenu(HelpMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "Documentation";
+        m_menuManagerInterface->RegisterMenu(HelpDocumentationMenuIdentifier, menuProperties);
```

Comment:
comment: It's cool these will be able to be moved out to another place in future too so the everything doesn't have to happen centrally as before (similar to what @cgalvan mentioned above)